neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
```

Comment:
This should be the same (assuming that trunk is tangent to the sphere of course), so I don't see any reason to distinguish them.